examples/hello_world/petastorm_dataset/generate_petastorm_dataset.py

```diff
@@ -40,7 +40,7 @@ def row_generator(x):
             'array_4d': np.random.randint(0, 255, dtype=np.uint8, size=(4, 128, 30, 3))}
 
 
-def generate_petastorm_dataset(output_url='file:///tmp/hello_world_dataset'):
+def generate_petastorm_dataset(output_url='file:///tmp/hello_world_dataset', partitioned=False):
```

Comment:
Would suggest to keep the hello world example as minimal as possible. Partitioning is not a trivial feature and may confuse. Would suggest to copy this code into the test file itself.

Also, having a test depend on an example code is unexpected (unless the test is checking the example)

Reply:
Got it; I was attempting to minimize code replication - I'll undo this change and copy it into the test case.